FirebaseCore/Extension/FIRHeartbeatLogger.h

```diff
@@ -17,7 +17,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 #ifndef FIREBASE_BUILD_CMAKE
-@class FIRHeartbeatsPayload;
+@import FirebaseCoreInternal;
```

Comment:
The reason the API used in the above file wasn't previously available to Swift was that the API signatures referenced forward declared types. This doesn't work well (hence, this [proposal](https://forums.swift.org/t/pitch-importing-forward-declared-objective-c-classes-and-protocols/61926)), and the way around it is to import the type's module instead of forward declaring it. Then, when the header is parsed for it's Swift-compatible API, all API shows in Swift.

Reply:
The unfortunate part is that `FirebaseCoreExtension` now exports `FirebaseCoreInternal`, but that seems unavoidable.

Reply:
FirebaseCoreInternal is already a dependency of FirebaseCore, so in practice should have no impact.